docs/examples/include.rst

```diff
@@ -0,0 +1 @@
+.. include:: thingtoinclude.rst
```

Comment:
Shouldn't this be the attention box and content?

Reply:
apologies I didn't commit all the changes *facepalm* should be ok now!